lib/api/wasm_c.cpp

```diff
@@ -518,6 +537,684 @@ WASM_DECLARE_C_GET_ATTR(exporttype, wasm_externtype_t *, type, static, nullptr,
 
 /// <<<<<<<< wasm_exporttype_t functions <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
 
+#define UNUSED(x) (void)(x)
+
+///////////////////////////////////////////////////////////////////////////////
+// Runtime Values
+
+// References
+
+struct wasm_ref_t : public wasm::Ref {};
+
+#define WASM_DEFINE_REF_BASE(name, Name)                                       \
```

Comment:
Move the macro to the top, as the same place as the other macros in anonymous namespace.